playwright-tests/tests/stake-delegation/stake-delegation.spec.js

```diff
@@ -1301,23 +1305,6 @@ test.describe("Insufficient balance ", function () {
     ).toBeVisible();
   });
 
-  test("Should throw error with approve vote when account has insufficient near balance", async ({
```

Comment:
Why is this test removed?

Reply:
actually we don't allow user to add a stake/unstake request, for more than the available/staked amount, so I removed the checks from the 'Approve' action, since it won't happen, and other dao frontend doesn't have UI to add staking request.